Explorer/Assets/DCL/Navmap/StreetViewController.cs

```diff
@@ -50,6 +54,13 @@ public void Deactivate()
             view.gameObject.SetActive(false);
         }
 
+        public void Animate(int triggerId)
+        {
+            view.gameObject.SetActive(triggerId == IN);
+        }
+
+        public void ResetAnimator() { }

```

Comment:
idem here xD

Reply:
Ah I see this ones dont have an animator xD its just set active xD ignore me then xDD

Reply:
xD